src/storage-controller/src/lib.rs

```diff
@@ -469,7 +469,7 @@ where
         migrated_storage_collections: &BTreeSet<GlobalId>,
     ) -> Result<(), StorageError<Self::Timestamp>> {
         self.migrated_storage_collections
-            .clone_from(migrated_storage_collections);
+            .extend(migrated_storage_collections.iter().cloned());
```

Comment:
@jkosh44 Does this look reasonable? It seems like before the first call to `create_collections` would clear `self.migrated_storage_collections`, which I don't think is intentional?

Reply:
Oh yikes, yeah this definitely seems reasonable. The old behavior looks like a mistake.